dts/arm/st/h7/stm32h723.dtsi

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2020 Alexander Kozhinov
+ * AlexanderKozhinov@yandex.com
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/h7/stm32h7.dtsi>
+
+/ {
+	/* DTCM memory directly coppled to CPU */
+	dtcm: memory@20000000 {
+		compatible = "arm,dtcm";
+		reg = <0x20000000 DT_SIZE_K(128)>;
+	};
+
+	/* AXI SRAM in D1 domain (AXI bus) */
+	sram0: memory@24000000 {
```

Comment:
sram0 and dtcm should only be the memories defined as these are the supported ones. Others have specific uses for which Zephyr is not ready (or has not been tested)

Reply:
I would leave at least current memory areas, since they are not a showstoppers.

Reply:
As mentioned, except dtcm and sram0 others are currently not supported or not tested, so they should be removed.

Reply:
Done